arch/x86/core/intel64/cpu.c

```diff
@@ -28,6 +28,11 @@ extern uint8_t _exception_stack1[];
 extern uint8_t _exception_stack2[];
 extern uint8_t _exception_stack3[];
 
+extern uint8_t _nmi_stack[];
```

Comment:
please prefix with `z_x86`, `_` prefix is deprecated (MISRA-C violation)

Reply:
Fixed for `_nmi_stack` as well for `_exception_stack`